samples/app-dev/API-Development/REST/allOf_oneOf-anyOf_construct_schema/readme.md

```diff
@@ -0,0 +1,125 @@
+This sample demonstrates some of the combinator keywords features present in the FLOGO ReceiveHTTPMessage trigger and InvokeRestService activity. Features which are covered in these sample apps are:
+
+## ReceiveHTTPMessage trigger
+1. Path, query and header parameters in the REST trigger.
+2. Configure multiple response code in REST trigger.
+3. Response Headers in REST trigger.
+4. App level schema in Request Schema and mapper activities.
+5. Multiple branching for each Response code.
+6. ConfigureHTTPResponse activity to map corresponding response schema and header with Return activity.
+
+## InvokeRestService activity
+1. Configuring InvokeRest activity with the API Spec of the producer REST service.
+2. Path, query, header parameters and Request and Response schema will be auto-populated.
+3. App property for the URL field which can be overridden at runtime as per the request URL.
+
+## Import the sample
+1. Download the sample json files i.e., *AllOf_OneOf_AnyOf_Keywords_APISpec_Service.json* and *Invoke_AllOf_OneOf_AnyOf_Keywords_APISpec_Service.json* 
+Rest triggers in the service app have been configured with Open API Specs 3.0 files. These API Specs are available in the *OpenAPISpecs* folder.
+
+2. Create a new empty app.
+![Create an app](../../import-screenshots/2.png)
+
+3. On the app details page, select Import app.
+![Select import](../../import-screenshots/3.png)
+
+4. Browse on your machine or drag and drop the .json file for the app that you want to import.
+![Import your sample](../../import-screenshots/4.png)
+
+5. Click Upload. The Import app dialog displays some generic errors and warnings as well as any specific errors or warnings pertaining to the app you are importing. It validates whether all the activities and triggers used in the app are available in the Extensions tab.
+![The Import app dialog](../../import-screenshots/5.png)
+
+6. You have the option to import all flows from the source app or selectively import flows.
+
+7. Click Next. If you had not selected a trigger in the previous dialog, the flows associated with that trigger are displayed. You have the option to select one or more of these flows such that the flows get imported as blank flows that are not attached to any trigger. By default, all flows are selected. Clear the check box for the flows that you do not want to import. If your flow(s) have subflows, and you select only the main flow but do not select the subflow, the main flow gets imported without the subflow. Click Next.
+
+
+## Understanding the configuration
+After importing the 'AllOf_OneOf_AnyOf_Keywords_APISpec_Service', we can see that multiple response codes have been configured in the Rest trigger. Few Response codes have Response headers as well. While adding schema in Response header and Response body we need to make sure that both are in JSON data or JSON Schema format.
```

Comment:
Not applicable
Few Response codes have Response headers as well. While adding schema in Response header and Response body we need to make sure that both are in JSON data or JSON Schema format.